src/metabase/cmd/load_from_h2.clj

```diff
@@ -100,6 +102,8 @@
    Collection
    CollectionRevision
    DashboardFavorite
+   FieldFingerprint
+   TableFingerprint
```

Comment:
keeping this list alphabetized will make things a little easier, help us avoid duplicates, etc.